Remotion/Core/Core/Core.Core.csproj

```diff
@@ -20,6 +20,7 @@
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.15" />
     <PackageReference Remove="JetBrains.Annotations" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="8.0.0" />
```

Comment:
Since we are only adding things to Core.Extensions, this should probably be added there, not in `Core.Core` for now.